README.md

```diff
@@ -1,66 +1,86 @@
-# duniverse -- the spice of OCaml build life
+# opam-monorepo
 
-This is an experimental vendoring system for Dune.  Not for public consumption
-just yet.
+`opam-monorepo` is an opam plugin designed to assemble standalone dune workspaces
+with your projects and all of their dependencies, letting you build it all from scratch
+using only `dune`.
 
-Please see the manual pages for an explanation of how to use the tool:
+## Installation
 
-- [duniverse](#duniverse): the overall usage
-- [duniverse init](#duniverse-init)
-- [duniverse pull](#duniverse-pull)
+You can simply install it via opam in your current switch by running:
 
-Contact `@avsm` if you have any queries.
+```
+opam install opam-monorepo
+```
 
+Note that once it is installed you can invoke it as an opam command:
+```
+opam monorepo <subcommand> ...
+```
 
-# Manual Pages
+Note that opam knows about available plugins and will offer you to install opam-monorepo
+if you try to invoke it without having installed it beforehand.
 
-## duniverse
+## Usage
 
+The basic usage for opam-monorepo is to start by running the following command from the root
+of your project:
+```
+opam monorepo lock
 ```
-NAME
```

Comment:
In opam-compiler I added [some rules](https://github.com/ocaml-opam/opam-compiler/blob/master/doc/create.txt) to dump the generated docs into text files, I found that was a nice tradeoff between writing markdown docs and having people install the program before they're able to read the docs.

Reply:
I think a proper readme will look much nicer than a manpage ever will so I'd rather maintain them separately to be honest.

Reply:
Hmm ok I see what you mean, yeah we could eventually add that separately in a subsequent PR!